assets/router/daemonset.yaml

```diff
@@ -0,0 +1,44 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: router
+  labels:
+    app: router
+spec:
+  selector:
+    matchLabels:
+      app: router
+  template:
+    metadata:
+      labels:
+        app: router
+    spec:
+      serviceAccountName: router
+      nodeSelector:
+        "node-role.kubernetes.io/infra": "true"
```

Comment:
Presumably the factory may override this in the future depending on `ClusterIngress.Spec.NodePlacement`.

Reply:
I think that makes sense